### PR TITLE
Refactor styles

### LINKED ISSUE #398: Improve styles codebase
The currect styles have multiple problems

- [x] Drop Sass for Stylus or PostCSS to avoid non-js binaries [open discussion]
- [x] Remove all nesting in selectors
- [ ] Stricter lint rules
  - [ ] More variables/mixins
  - [ ] 6 digits colors
  - ...
- ...

### LINKED ISSUE #468: Hovering the progress part of the playing bar show the tooltip at a wrong location

### PR DESCRIPTION
**Pull request by martpie**
- [x] Fix #398 Refactor styles
- [x] Implement #437 Kill Bootstrap 
- [x] Fix #468 Tooltip not correctly positioned
- [ ] Last refinements

### CODE REVIEW

File: src/ui/actions/SettingsActions.ts [hunk 1]
```diff
@@ -6,18 +6,32 @@ import types from '../constants/action-types';
...
   silentFail?: boolean
 };
 
-const checkTheme = () => {
+/**
+ * Apply theme
+ */
+export const applyTheme = (theme: Theme) => {
+  // TODO think about variables validity
+  let root = document.documentElement;
```
Comment (YurySolovyov) [inline src/ui/actions/SettingsActions.ts]: const?

File: src/ui/elements/Heart.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import * as React from 'react';
+
+import * as styles from './Heart.css';
+
+const Heart: React.FC = () => (
+  <span className={styles.heart}>♥</span>
```
Comment (YurySolovyov) [inline src/ui/elements/Heart.tsx]: why not just inline it?

Comment (martpie) [reply src/ui/elements/Heart.tsx]: For consistency I guess

**Comment by martpie** [comment]
Last refinements can probably be done on `master`

**Pull request merged**